### PR TITLE
Small clean

### PR DESCRIPTION
**Pull request by stephomi**

### CODE REVIEW

File: sources/osgViewer/eventProxy/DeviceOrientation.js [hunk 1]
```diff
@@ -49,6 +45,9 @@ DeviceOrientation.prototype = {
         if ( !this._enable )
             return false;
 
+        if ( !this._deviceOrientation )
```
Comment (cedricpinson) [inline sources/osgViewer/eventProxy/DeviceOrientation.js]: could you explain why this has been added ?

Comment (stephomi) [inline sources/osgViewer/eventProxy/DeviceOrientation.js]: Don't use the default orientation if the device don't support orientation
api.

File: sources/osg/Polytope.js [hunk 2]
```diff
@@ -334,9 +334,9 @@ Polytope.prototype = MACROUTILS.objectInherit( Object.prototype, {
      * calculated elsewhere then use transformProvidingInverse() instead.
      * See http://www.worldserver.com/turk/computergraphics/NormalTransformations.pdf*/
     transform: function ( matrix ) {
-        var inverse = new Matrix();
-        inverse.invert( matrix );
-        this.transformProvidingInverse( inverse );
+        var inv = Matrix.create();
```
Comment (stephomi) [inline sources/osg/Polytope.js]: @jtorresfabra this function was never called I guess (Matrix is not a function)

Not sure if we can re-use matrix or not.

Comment (Kuranes) [inline sources/osg/Polytope.js]: *cough*cough* seems that might be me the code author, and afaicr the code can be removed now

**Pull request merged**